tests/functional/spec/progress_value_source_hub_with_multiple_dependencies.spec.js

```diff
@@ -0,0 +1,126 @@
+import ListCollectorPage from "../generated_pages/list_collector_primary_person/list-collector.page.js";
```

Comment:
The tests in this file doesn't seem to have anything to do with progress source?